src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * An association class between person and tag. Maintained by the address book object.
+ * 
+ * @isActive Shows if the tagging is added or deleted.
+ */
+public class Tagging {
+		
+	Person person;
+	Tag tag;
+	boolean isActive;
```

Comment:
Probably can give more meaning as to what active means?  Alternatively you can use a `tagStatus` variable and use two `enum` values `added` and `deleted`